docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -233,408 +122,121 @@ is used.
     - Enable to allow user to call ``register_table`` procedure
     - ``false``
 
-ORC format configuration
-^^^^^^^^^^^^^^^^^^^^^^^^
-
-The following properties are used to configure the read and write operations
-with ORC files performed by the Iceberg connector.
-
-.. list-table:: ORC format configuration properties
-  :widths: 30, 58, 12
-  :header-rows: 1
-
-  * - Property name
-    - Description
-    - Default
-  * - ``hive.orc.bloom-filters.enabled``
-    - Enable bloom filters for predicate pushdown.
-    - ``false``
-
-Parquet format configuration
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The following properties are used to configure the read and write operations
-with Parquet files performed by the Iceberg connector.
-
-.. list-table:: Parquet format configuration properties
-    :widths: 30, 50, 20
-    :header-rows: 1
-
-    * - Property Name
-      - Description
-      - Default
-    * - ``parquet.max-read-block-row-count``
-      - Sets the maximum number of rows read in a batch.
-      - ``8192``
-    * - ``parquet.optimized-reader.enabled``
-      - Whether batched column readers should be used when reading Parquet files
-        for improved performance. Set this property to ``false`` to disable the
-        optimized parquet reader by default. The equivalent catalog session
-        property is ``parquet_optimized_reader_enabled``.
-      - ``true``
-    * - ``parquet.optimized-nested-reader.enabled``
-      - Whether batched column readers should be used when reading ARRAY, MAP
-        and ROW types from Parquet files for improved performance. Set this
-        property to ``false`` to disable the optimized parquet reader by default
-        for structural data types. The equivalent catalog session property is
-        ``parquet_optimized_nested_reader_enabled``.
-      - ``true``
-
-.. _iceberg-authorization:
-
-Authorization checks
-^^^^^^^^^^^^^^^^^^^^
-
-You can enable authorization checks for the connector by setting
-the ``iceberg.security`` property in the catalog properties file. This
-property must be one of the following values:
+Metastores
+----------
 
-.. list-table:: Iceberg security values
-  :widths: 30, 60
-  :header-rows: 1
+Iceberg natively uses the concept of a catalog. The subsection titles in this
```

Comment:
How about this as a replacement for the whole paragraph.

The Iceberg table format manages most metadata in metadata files in the object storage itself. A small amount of metadata however still requires the use of a metastore. In the Iceberg ecosystem, these smaller metastores are called Iceberg metadata catalogs, or just catalogs. The following sections detail the available options of metadata catalogs you can use.

Reply:
Met you half way.